src/main/java/com/conveyal/datatools/manager/controllers/api/ProjectController.java

```diff
@@ -140,7 +138,7 @@ private static Project deleteProject(Request req, Response res) {
         Project project = requestProjectById(req, "manage");
         boolean successfullyDeleted = project.delete();
         if (!successfullyDeleted) {
-            haltWithMessage(req, 400, "Did not delete project.");
+            haltWithMessage(req, 500, "Did not delete project.", new Exception("Delete unsuccessful"));
```

Comment:
It doesn't seem great to be just creating arbitrary exceptions here that are not really tied back to the "failing" code. Do we gain anything by having a bugsnag report that points back here?

Reply:
I don't think it is possible to create a Bugsnag report without an exception.   So I think what we'd gain here is receiving a Bugsnag report that the user received a 500 error from the server.